app/src/main/res/values/preference_keys.xml

```diff
@@ -9,6 +9,7 @@
     <string name="pref_key_privacy_block_analytics" translatable="false"><xliff:g id="preference_key">pref_privacy_block_analytics</xliff:g></string>
     <string name="pref_key_privacy_block_social" translatable="false"><xliff:g id="preference_key">pref_privacy_block_social</xliff:g></string>
     <string name="pref_key_privacy_block_other" translatable="false"><xliff:g id="preference_key">pref_privacy_block_other</xliff:g></string>
+    <string name="pref_key_privacy_block_abpindo" translatable="false"><xliff:g id="preference_key">pref_privacy_block_abpindo</xliff:g></string>
```

Comment:
This may exceed the length limit that is enforced in a patch that @cnevinc just landed. Could be a problem after rebasing.

Reply:
What's the length limitation? 40char? I guess analytics will exceed the boundary first?
Maybe we should write a custom lint to ensure the char length?